.circleci/config.yml

```diff
@@ -1,9 +1,48 @@
 version: 2.1
 jobs:
-  build:
+  Build Development Images:
     environment:
-      SITENAME: "CircleCI 2.1"
-    machine: true
+      SITENAME: CircleCI
```

Comment:
Maybe it's the different site name?  Things before today are `CircleCI 2.1` while this branch submits as `CircleCI`?